bin/liquidator/src/liquidate.rs

```diff
@@ -541,7 +464,7 @@ impl<'a> LiquidateHelper<'a> {
     }
 
     async fn token_liq_bankruptcy(&self) -> anyhow::Result<Option<Signature>> {
-        if !self.health_cache.in_phase3_liquidation() || !self.health_cache.has_spot_borrows() {
+        if !self.health_cache.in_phase3_liquidation() || !self.health_cache.has_liq_spot_borrows() {
```

Comment:
nit: has_spot_borrows was more appropriate imo then has_liq_spot_borrows, it still only checks if there is any token with borrow, nothing more

Reply:
No, it's different now. It checks if there is a spot balance and the sum of the spot balance and all perp uhupnl (which is weighted!) is negative.